src/data/graphql/queries.graphql

```diff
@@ -388,7 +392,8 @@ query SubgraphActions($skip: Int!, $first: Int!, $colonyAddress: String!) {
       "TokensMinted(address,address,uint256)",
       "DomainAdded(address,uint256)",
       "ColonyMetadata(address,string)",
-      "ColonyFundsMovedBetweenFundingPots(address,uint256,uint256,uint256,address)"
+      "ColonyFundsMovedBetweenFundingPots(address,uint256,uint256,uint256,address)",
+      "ColonyRoleSet(address, address, uint256, uint8, bool)"
```

Comment:
Please remove the spaces between the args, to keep this consistent with the others

Reply:
If I do that, then the query won't catch the ColonyRoleSet events. The name of the query includes the white spaces

Reply:
If I do that, then the query won't catch the ColonyRoleSet events. The name of the query includes the white spaces